src/libs/OptionsListUtils.js

```diff
@@ -699,8 +700,13 @@ function getOptions(
         });
     }
 
+    let currentUserOption = _.find(allPersonalDetailsOptions, (personalDetailsOption) => personalDetailsOption.login === currentUserLogin);
+    if (searchValue && !isSearchStringMatch(searchValue, currentUserOption.searchText)) {
+        currentUserOption = null;
+    }
```

Comment:
Seems like it may be possible to not find `currentUserLogin` within the `allPersonalDetailsOptions`, or at least it is happening to me in dev:

<img width="1893" alt="image" src="https://github.com/Expensify/App/assets/87341702/c6d913ac-96ec-40b2-9e7e-2c0cfa219c91">

Reply:
Any key I type in the search just crashes

Reply:
Selectors were likely broken with the privacy changes to use accountIDs